docs/handbook/image-file-formats.rst

```diff
@@ -483,6 +483,13 @@ The :py:meth:`~PIL.Image.Image.save` method supports the following options:
 **exif**
     If present, the image will be stored with the provided raw EXIF data.
 
+**keep_colorspace**
```

Comment:
For me this is not clear name, since it affects only RGB to YCbCr conversion

Reply:
It prevents libjpeg from doing _any_ automatic colorspace conversion, and currently that's only relevant for RGB to YCbCr.  If you'd prefer, I can rename to e.g. `keep_rgb` and change the code to specifically only handle the RGB case.

Reply:
I think this is better than `keep_rgb` would be.